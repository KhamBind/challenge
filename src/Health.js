import React from 'react'

const Health = (props) => {
    return(
        <div className="progress-bar">
            <div className="progress-inbar" 
                style={{
                    width: props.value +'%', 
                    backgroundColor: props.value <= 50 ? '#5cb85c': props.value <= 70 ? '#f0ad4e' : '#f0ad4e'
                }}>
            </div>
        </div>
    );
}

export default Health;