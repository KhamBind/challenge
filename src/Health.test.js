import React from 'react'
import renderer from 'react-test-renderer'
import Health from './Health'

describe('Health', () => {
    let component = null;

    it('renders correctly', () => {
        component = renderer.create(<Health />);
    });

    it('matches snapshot', () => {
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});