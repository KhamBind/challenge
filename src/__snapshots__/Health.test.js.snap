// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Health matches snapshot 1`] = `
<div
  className="progress-bar"
>
  <div
    className="progress-inbar"
    style={
      Object {
        "backgroundColor": "#f0ad4e",
        "width": "undefined%",
      }
    }
  />
</div>
`;
